src/containers/PackingOverview/RemindersPage/AddTodo/AddTodo.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const AddTodo = props => {
```

Comment:
> So this PR doesn't get too long, in my next PR I will work on cleaning up the RemindersPage so as not to contain so much code in one file.

In my experience, this often ends up not happening. Priorities change, people forget, etc. It would be a good idea if you did the cleanup first, _then_ added the new features. As a benefit, the new features and the reviewing of the new features both get easier.

Reply:
Unfortunately, I have to add these to the backlog due to time constraints. I feel the impact of this though so I'll be mindful of it going forward.